RELEASE_NOTES.md

```diff
@@ -13,8 +13,18 @@
 - A tutorial section and a getting started tutorial
 - In `OrderedRingBuffer`:
   - Rename `datetime_to_index` to `to_internal_index` to avoid confusion between the internal index and the external index.
+  - Add `index_to_datetime` method to convert external index to corresponding datetime.
   - Remove `__setitem__` method to enforce usage of dedicated `update` method only.
+- In `OrderedRingBuffer` and `MovingWindow`:
+  - Support for integer indices is added.
+  - Add `count_covered` method to count the number of elements covered by the used time range.
+
+
+
 
 ## Bug Fixes
 
 <!-- Here goes notable bug fixes that are worth a special mention or explanation -->
+
+- The `__getitem__` magic of the `MovingWindow` is fixed to support the same functionality
+  that the `window` method provides.
```

Comment:
Lines shouldn't be wrapped in the release notes because GitHub releases doesn't render the line breaks properly.

```suggestion
- The `__getitem__` magic of the `MovingWindow` is fixed to support the same functionality that the `window` method provides.
```